junos/resource_static_route_test.go

```diff
@@ -80,6 +100,26 @@ func TestAccJunosStaticRoute_basic(t *testing.T) {
 							"community.#", "1"),
 						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_instance",
 							"community.0", "no-advertise"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"routing_instance", "testacc_staticRoute"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"preference", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"metric", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"qualified_next_hop.#", "1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"qualified_next_hop.0.next_hop", "2001:db8:85a4::1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"qualified_next_hop.0.preference", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"qualified_next_hop.0.metric", "102"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"qualified_next_hop.0.interface", "st0.0"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"community.#", "1"),
+						resource.TestCheckResourceAttr("junos_static_route.testacc_staticRoute_ipv6_interface",
+							"community.0", "no-advertise"),
```

Comment:
Same comment as for ipv4. Just test `interface` argument

Reply:
Fixed